lib/iptables.js

```diff
@@ -283,12 +283,12 @@ function exec_next() {
 			if (--tries_left <= 0) throw error + ' / Cannot run ' + cmd;
```

Comment:
Just noting: here, if all the retries fail, this will still throw here. In the case of additions, the throw will prevent the callback to run, and thus will prevent the old rules to be deleted. In a (future) other PR, the callback system of this file should be updated such that all calls are garanteed to callback, and errors can be reported in the callbacks (using the "standard" signature ```callback(error, data)```)